### DISCUSSION TITLE
Create assignments via an API

### DISCUSSION THREAD

**Discussion by buc-htl** [community/community#67006] [GitHub Education] [answered]
### Hi everyone,

Is there a possibility to create assignments automatically for a classroom?

Thanks for your inputs!

**Comment by seyLu** [comment, marked as answer]
Hi @buc-htl,

Apparently it is still under development, the github api for classrooms. You can check out this recent [issue](https://github.com/github/gh-classroom/issues/33), where a github staff commented about its status.
